functions/src/Controller/Member.ts

```diff
@@ -1,28 +1,29 @@
-import { Member } from '../Models/Member';
-import * as functions from 'firebase-functions';
-import { db } from '../index';
```

Comment:
Hey Jeff, you removed db in here, was that an accident or on purpose?

Reply:
Jeff please bring back db so we can maintain the db as our firestore variable throughout all of our code.